rtengine/metadata.cc

```diff
@@ -312,6 +313,13 @@ void Exiv2Metadata::saveToImage(const Glib::ustring &path, bool preserve_all_tag
     }
 
     dst->exifData()["Exif.Image.Software"] = "RawTherapee " RTVERSION;
+
+    std::time_t t = std::time(nullptr);
+    char mbstr[100];
+    if (std::strftime(mbstr, sizeof(mbstr), "%Y:%m:%d %H:%M:%S", std::localtime(&t))) {
+    dst->exifData()["Exif.Image.DateTime"] = mbstr;
```

Comment:
Indentation

Reply:
```
if (std::strftime(mbstr, sizeof(mbstr), "%Y:%m:%d %H:%M:%S", std::localtime(&t))) {
    dst->exifData()["Exif.Image.DateTime"] = mbstr;
```

You mean like this? (Github automatically added the indentation when I pasted the code)